# In-Class Activity (July 9, 2026)

In [1]:
library(tidyverse)
library(moderndive)

── Attaching core tidyverse packages ─────────────────────────────────────── tidyverse 2.0.0 ──
✔ dplyr     1.2.1     ✔ readr     2.2.0
✔ forcats   1.0.1     ✔ stringr   1.6.0
✔ ggplot2   4.0.3     ✔ tibble    3.3.1
✔ lubridate 1.9.5     ✔ tidyr     1.3.2
✔ purrr     1.2.2     
── Conflicts ───────────────────────────────────────────────────────── tidyverse_conflicts() ──
✖ dplyr::filter() masks stats::filter()
✖ dplyr::lag()    masks stats::lag()
ℹ Use the conflicted package (<http://conflicted.r-lib.org/>) to force all conflicts to become errors


In [2]:
wage_data <- read.table("data/wage.txt", header=TRUE)
head(wage_data)

,education,south,sex,experience,union,wage,age,race,occupation,sector,marr
,<int>,<int>,<int>,<int>,<int>,<dbl>,<int>,<int>,<int>,<int>,<int>
1,8,0,1,21,0,5.10,35,2,6,1,1
2,9,0,1,42,0,4.95,57,3,6,1,1
3,12,0,0,1,0,6.67,19,3,6,1,0
4,12,0,0,4,0,4.00,22,3,6,0,0
5,12,0,0,17,0,7.50,35,3,6,0,1
6,13,0,0,9,1,13.07,28,3,6,0,0


# A: Is the relationship between WAGE and EDUCATION statistically significant?

In [3]:
wage_model <- lm(wage ~ education, data = wage_data)
get_regression_table(wage_model)

term,estimate,std_error,statistic,p_value,lower_ci,upper_ci
<chr>,<dbl>,<dbl>,<dbl>,<dbl>,<dbl>,<dbl>
intercept,-0.746,1.045,-0.714,0.476,-2.800,1.308
education,0.750,0.079,9.532,0.000,0.596,0.905


The p-value for education which is the B1_hat value is 0. Therefore, we can conclude that the relationship between wage and education is statistically significant since this value is below 0.05 (the significance level).

# B: Find a 95% confidence interval for β1 (slope in the simple linear regression model). Is the result consistent with (a)?

In [4]:
get_regression_table(wage_model)

term,estimate,std_error,statistic,p_value,lower_ci,upper_ci
<chr>,<dbl>,<dbl>,<dbl>,<dbl>,<dbl>,<dbl>
intercept,-0.746,1.045,-0.714,0.476,-2.800,1.308
education,0.750,0.079,9.532,0.000,0.596,0.905


The 95% confidence interval for beta_1 is (0.596, 0.905). Since this range does not include 0, we can reject the null hypothesis and claim that the results are statistically significant at a 5% significance level.

# C: What is the relationship between the 95% confidence interval for β1 and the z-statistic (and the corresponding p-value) from R output?

The z-statistic exceeds 1.96 and the p-value is below 0.05 in this case which is equivalent to the confidence interval not including 0 which would mean statistical significance. At Z=1.96 and p=0.05 would be the threshold where 0 would be on the 95% confidence interval. Since Z=9.532 and p=0 for B1, we can determine from the Z-statistic and p-value that the results are statistically significant.